requirements.txt

```diff
@@ -1,4 +1,5 @@
 beautifulsoup4
+praw
```

Comment:
Not in alphabetical order.  We are going to need multiple algorithms that use praw to justify its addition to this small list.

Reply:
Fixed in alphabetical order. Finding a way to do this without praw.